system/Database/BaseConnection.php

```diff
@@ -392,7 +395,13 @@ public function initialize()
 			// We still don't have a connection?
 			if (! $this->connID)
 			{
-				throw new DatabaseException('Unable to connect to the database.');
+				$nl2br = is_cli() ? PHP_EOL : '<br/>';
```

Comment:
I don't like this class caring about CLI versus web. I understand this is an exception in the making but can we find another way to do this?

Reply:
Exceptions are already caught and have templates specific to html or cli, right? So shouldn't be a need for it here unless I'm missing something. Seems a better choice is to use line breaks here, and make sure the exception templates deal with it correctly. That's the proper place for this, I believe. Otherwise, it's just a band-aid on a different problem and we try to keep band-aids away from the core framework. :)